mass-deployer/README.md

```diff
@@ -0,0 +1,59 @@
+# mass-deployer
+
+Mass Deplyer tool designed to automate mass deployment of batchs of VMs on ThreeFold Grid.
+
+## Features
+
+-   **Mass Deployment:** Deploy groups of vms on ThreeFold Grid simultaneously.
+-   **Customizable Configurations:** Define Node groups, VMs groups and other configurations through YAML or JSON file.
+
+## Usage
+1.  First [download](#download) mass-deployer binaries.
+2.  Create a new configuration file.
+
+```yaml
+node_groups:
+  - name: example-group
+    nodes_count: 3
+    free_cpu: 8
+    free_mru: 16384
+    # ... other fields
+
+vms:
+  - name: example-vm
+    vms_count: 2
+    node_group: example-group
+    cpu: 2
+    mem: 4096
+    flist: example-flist,
+    entry_point: /sbin/zinit init
+    # ... other fields
+
+sshkey: example-ssh-key
```

Comment:
We have agreed to have the ssh key with its name (to allow more than one ssh key)

Reply:
so it's going to be a list of sshkeys?